apps/explorer/src/app/routes/oracles/Oracles.graphql

```diff
@@ -49,6 +49,37 @@ fragment ExplorerOracleDataSource on OracleSpec {
           }
           ... on DataSourceDefinitionExternal {
             sourceType {
+              ... on EthCallSpec {
+                abi
+                args
+                method
+                requiredConfirmations
+                address
+                normalisers {
+                  name
+                  expression
+                }
+                trigger {
```

Comment:
**praise (non-blocking)**: I was going to ask if it was a problem that you were only fetching details of `EthTimeTrigger` triggers, but I see in the schema that's the only valid option at this point 👍